README.md

```diff
@@ -9,7 +9,7 @@ Assuming you have our [Vagrant box](https://github.com/uw-midsun/box) installed:
 
 ```bash
 # Clone the repo
-git clone https://github.com/uw-midsun/firmware.git firmware
+git clone https://github.com/uw-midsun/firmware_xiv.git firmware_xiv
 cd firmware
```

Comment:
should also be `cd firmware_xiv` here I think?

Reply:
oh that's true, fixed